src/backend/distributed/master/master_create_shards.c

```diff
@@ -88,7 +93,7 @@ master_create_worker_shards(PG_FUNCTION_ARGS)
  */
 void
 CreateShardsWithRoundRobinPolicy(Oid distributedTableId, int32 shardCount,
-								 int32 replicationFactor)
+								 int32 replicationFactor, bool emptyTable)
```

Comment:
I think the name `emptyTable` holds no particular relevance to this function. Maybe `useExclusiveConnections`.